hw/ip/prim_generic/lint/prim_generic_clock_div.waiver

```diff
@@ -18,3 +18,27 @@ waive -rules CLOCK_USE -location {prim_generic_clock_div.sv} -regexp {'clk_i' is
 
 waive -rules SAME_NAME_TYPE -location {prim_generic_clock_div.sv} -regexp {'ResetValue' is used as a parameter here, and as an enumeration value at} \
       -comment "Reused parameter name."
+
+waive -rules CLOCK_DRIVER -location {prim_generic_clock_div.sv} -regexp {'gen_div2.q_p' is driven by instance 'gen_div2\^u_div2' of module 'prim_flop', and used as a clock 'clk_i'} \
+      -comment "A 'prim_flop' is used together with a 'prim_clock_inv' to implement a division by 2"
+
+waive -rules CLOCK_USE -location {prim_generic_clock_div.sv} -regexp {'gen_div2.q_n' is connected to 'prim_flop' port 'd_i\[0\]', and used as a clock 'clk_i'} \ 
```

Comment:
Looks like an extra space got placed at the end of this line.

Reply:
Thanks, fixed.